server/test/auth.test.js

```diff
@@ -37,7 +37,7 @@ describe("Test for all auth endpoints", () => {
         .post("/auth/register")
         .send(newUser)
         .end((err, res) => {
-          res.should.have.status(409);
+          res.should.have.status(400);
```

Comment:
@DestinTech Change the error code back to 409. [Here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409) is the reference.

Reply:
I have changed that for you now!